### PR TITLE
Update french translation GUI

### PR DESCRIPTION
**Pull request by Nilvus**
An update of french GUI, especially last tagging improvements.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -13021,208 +13092,297 @@ msgstr[1] ""
...
+msgstr "effacer texte"
+
+#: ../src/libs/tagging.c:2649
 msgid ""
-"related tags,\n"
+"tag dictionary,\n"
 "double-click to attach,\n"
-"right-click for other actions on selected tag"
+"right-click for other actions on selected tag,\n"
+"Ctrl-wheel scroll to resize the window"
```
Comment (TurboGit) [inline po/fr.po]: No C capital letter here.

File: po/fr.po [hunk 2]
```diff
@@ -13021,208 +13092,297 @@ msgstr[1] ""
...
 msgstr ""
 "mots-clés liés,\n"
 "double-clic pour attacher\n"
-"clic-droit pour copier l'entrée"
+"clic-droit pour afficher le menu des actions du mot-clé sélectionné\n"
+"Ctrl+molette souris pour redimensionner la fenêtre"
```
Comment (TurboGit) [inline po/fr.po]: Same here.

Comment (TurboGit) [inline po/fr.po]: Same here.

File: po/fr.po [hunk 3]
```diff
@@ -13021,208 +13092,297 @@ msgstr[1] ""
...
+"double-click to detach\n"
+"right-click for other actions on attached tag,\n"
+"Ctrl-wheel scroll to resize the window"
 msgstr ""
-"mots-clés attachés,\n"
-"double-clic pour détacher"
+"mots-clés liés,\n"
+"double-clic pour attacher\n"
+"clic-droit pour afficher le menu des actions du mot-clé sélectionné\n"
+"Ctrl+molette souris pour redimensionner la fenêtre"
```
Comment (TurboGit) [inline po/fr.po]: No C capital letter here. Same above. And maybe in other places.

File: po/fr.po (hunk 1 above)
Comment (Nilvus) [reply po/fr.po]: I've only respected how it was written in english. If you see the source, in english it is also written with a capital letter. I change that on my translation. For english one, I let you change that.

File: po/fr.po (hunk 3 above)
Comment (Nilvus) [reply po/fr.po]: Normally no other places as I have just respect how the source was written. And as I remember, that was the only ones.

**Comment by TurboGit** [comment]
Thanks.

File: po/fr.po (hunk 3 above)
Comment (TurboGit) [reply po/fr.po]: Note that you don't have to respect the casing of the standard English strings and when you see such casing do not hesitate to fix the sources directly as I have done just now.

**Pull request merged**